bin/snapshot-backups/downsync-cms

```diff
@@ -22,12 +22,28 @@ then
     exit 1
 fi
 
+SCRIPT_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+if [ -f $SCRIPT_DIR/includes ]; then
+  . $SCRIPT_DIR/includes
+fi
+
 # just testing?
 if [ x$1 == x"--dryrun" ]; then
   export echo=echo
   shift
 fi
 
+needsDisabled=0
```

Comment:
I'd like this variable name to include "spaceSSH" or just "SSH". `needsSpaceSSHDisabled`